AUTHORS.rst

```diff
@@ -25,3 +25,4 @@ considered "The PyVista Developers":
 - Eric Larson, (`@larsoner <https://github.com/larsoner>`_)
 - Matthew Flamm, (`@MatthewFlamm <https://github.com/MatthewFlamm>`_)
 - Darik Gamble, (`@darikg <https://github.com/darikg>`_)
+- dcbr, (`@dcbr <https://github.com/dcbr>`_)
```

Comment:
@dcbr, if you are comfortable sharing your name, please add update it here. I think this massive of a contribution immediately warrants your name on the collective work! cc @pyvista/developers

Reply:
Thanks a lot :D